### DISCUSSION TITLE
[off-topic]ish Straight Install to rootfs...?

### DISCUSSION THREAD

**Discussion by darkwater4213** [dnschneid/crouton#4929] [Q&A] [answered]
Having disabled rootfs write protection (I need to get the bootloader fixed, though), I was wondering if it would be possible (or even viable) to essentiolly overwrite ChromeOS with a proper Linux installation. Not a chroot that kills ChromeOS (though I have tried that). I want ChromeOS gone entirely. I would like to use a... _more recent_ version of Ubuntu than Xenial (sorry, but that's just facts) so that I can use things like Node and Spotify (both of which depend on newer glibc versions than xenial has access to... :C ). But there comes two more problems: if it's so simple (knock on wood), why isn't this supported? And second, could I do this with other versions of Linux (say, Zorin OS, or Parrot, or Arch or something)? I can't boot the ISO version of Zorin (even with `<C-l>`) and I can't imagine that using a read-only version of Linux is a good way to set up a daily driver... Any thoughts?

**Comment by DennisLfromGA** [comment, marked as answer]
@darkwater4213,

mr.chromebox offers custom **UEFI firmware** for some ChromeOS devices, see the link below:

- https://wiki.mrchromebox.tech/Supported_Devices

If your Chromebook is a 'supported device' you can install custom firmware and completely replace ChromeOS with a Linux distro of your choice or optionally, other OSes. Note though that not everything may work so you may have to use it as is or find a fix for the issue.

If you do run into issues it would be best to seek support on one of the mr.chromebox sites, see below:

- https://mrchromebox.tech/#support


*Hope this helps,*
-**DennisLfromGA**

**Reply by darkwater4213** [reply]
Thanks! I discovered _exactly_ this (no joke) right after you replied. (Didn't see this until just now though) I have since done this and am replying from a full-functioning Linux machine XD. I installed GalliumOS. I know that it's an older distro, but there are a lot of features I like and I'm fine with being stuck on Bionic. I'll probably install Zorin OS or even Arch (lol) later, but I'm fine with this for now. Thanks anyway!
